modules/nw-osp-loadbalancer-limitations.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+// * networking/load-balancing-openstack.adoc
+// For thinking and reviewing, adding to networking/load-balancing-openstack.adoc
+
+:_content-type: CONCEPT
+[id="nw-osp-loadbalancer-limitations_{context}"]
+= Limitations of load balancer services
+
+{product-title} clusters on {rh-openstack-first} use Octavia to handle load balancer services. As a result of this choice, such clusters have a number of functional limitations.
+
+{rh-openstack} Octavia has two supported providers: Amphora and OVN. These providers differ in terms of available features as well as implementation details. These distinctions affect load balancer services that are created on your cluster.
+
+include::modules/nw-osp-loadbalancer-etp-local.adoc[leveloffset=+1]
```

Comment:
I think maybe this is the reason that the HTML preview isn't working - it's my understanding that you can't include a module in another module, as is being done with the include on this line and the include below. I think you would need to go up to the assembly and include all of the modules there, using the leveloffset to create the different heading levels.

Reply:
@dulek This is correct. Can we pull this statement into the assembly?